### DISCUSSION TITLE
centering Sympy output in PDF

### DISCUSSION THREAD

**Discussion by PascalGitz** [quarto-dev/quarto-cli#6403] [Q&A] [answered]
### Description

Hello

Im using Quarto to create a documentation (book structure) with some calculations, done in Sympy. The goal is to create a PDF with all my calculation-output centered.

---
title: "Rayleigh-Quotienten"
format: pdf
execute:
  echo: false
  warning: false
---

```
import sympy as sp
from sympy.abc import *

test = a + B
display(test)
```

The code generates this latex code (inline math):

`$\displaystyle B + a$`

Is it possible to change it to this:
`$$\displaystyle B + a$$`

Or is there another approach to achive my goal ?

**Comment by rgaiacs** [comment]
@PascalGitz Thanks for the message. I believe this is a question to the SymPy community. Have you look at SymPy's configuration for printing LaTeX: https://docs.sympy.org/latest/tutorials/intro-tutorial/printing.html?

**Reply by PascalGitz** [reply, marked as answer]
Hello @rgaiacs 
Thanks for the quick response. You are absolutely right. I was able to solve my problem by using this printingsettings.

`sp.init_printing(use_latex='mathjax', latex_mode='equation*')`
